Net SDK Unit Tests/Unit Tests/Matchers/UserDefinedSegmentMatcherTests.cs

```diff
@@ -13,11 +15,16 @@ public class UserDefinedSegmentMatcherTests
         public void MatchShouldReturnTrueOnMatchingSegment()
         {
             //Arrange
-            var keys = new HashSet<string>();
+            var keys = new List<string>();
```

Comment:
I defined it in the interface, it receives a list but adds each of them in the concurrent dictionary when calling the AddKeys method